### PR TITLE
UI Fixes, Adding Spacing

### PR DESCRIPTION
**Pull request by jedtan**

### CODE REVIEW

File: src/components/Intro.js [hunk 1]
```diff
@@ -93,10 +90,11 @@ const useStyles = makeStyles((theme) => ({
     display: 'flex',
     direction: 'row',
     alignItems: 'center',
-    justifyContent: 'flex-start',
+    justifyContent: 'center',
     padding: theme.spacing(1),
```
Comment (kmjennison) [inline src/components/Intro.js:94]: The numbered circles should still align at the top even if the steps differ in the length of text ([mock here](https://www.figma.com/file/KCnKxjXCMFjYx1Mi6wZe3B/Tab-for-Seas-Web-Pages?node-id=12%3A442)):
https://www.chromatic.com/pullrequest?activeElementId=comparison-test-key-61b7dbed4af8a9ac39fc39ff-1200&appId=6154d1bab7cd94003aee223d&number=249&view=changes

**Review by kmjennison — CHANGES_REQUESTED** [review]

File: src/components/Intro.js (hunk 1 above)
Comment (amaliwanag2) [reply src/components/Intro.js:94]: easiest way to do this is use the 'align-self' property on the circle itself

Comment (amaliwanag2) [reply src/components/Intro.js:94]: also don't forget to adjust the circle text to typography ie <Avatar className={cx.numberCircle}>
              <Typography>1</Typography>
            </Avatar>

**Review by kmjennison — APPROVED** [review]
LGTM 🦊

**Pull request merged**